src/line/LineBot.js

```diff
@@ -171,4 +163,33 @@ module.exports = class {
     pickRandom(array) {
         return array && array.length ? array[Math.ceil(Math.random() * array.length) - 1] : null;
     }
+
+    registerMenu(event) {
+        if (_checkAuth(event, "registerMenu")) {
+            let param = this._extractParameter(event.message.text).split(' ');
+            let promise = this.menuManager.add(param[0], param[1]);
+
+            promise.then((message) => {
+                this.bot.replyText(message);
+            }).catch((message) => {
```

Comment:
```suggestion
            }).catch((errMessage) => {
```

같은 이름의 변수 `message`를 사용하니까 살짝 혼동되는데 이렇게 쓰는 건 어떨까요?

Reply:
그럴거면 차라리 `err`는 어떠세여

Reply:
동이함니다